router/src/components/route.rs

```diff
@@ -65,6 +76,11 @@ pub fn Route<E, F, P>(
     /// accessed with [`use_route_data`](crate::use_route_data).
     #[prop(optional, into)]
     data: Option<Loader>,
+    /// How this route should handle trailing slashes in its path.
+    /// Overrides any setting applied to [`crate::components::Router`].
+    /// Serves as a default for any inner Routes.
+    #[prop(optional)]
+    trailing_slash: Option<TrailingSlash>,
```

Comment:
Because `None` means "inherit trailing_slash setting from parent routes" (see `routes::inherit_settings` https://github.com/leptos-rs/leptos/pull/2217/files#diff-31f96b333e0c4d9391387d676c81a652b1a975d984ee51b3e2cbd1d4ecf8de8cR352)